proto/desmos/profiles/v3/query.proto

```diff
@@ -27,42 +27,20 @@ service Query {
   // have been made towards the user with the given address
   rpc IncomingDTagTransferRequests(QueryIncomingDTagTransferRequestsRequest)
       returns (QueryIncomingDTagTransferRequestsResponse) {
-    option (google.api.http).get =
-        "/desmos/profiles/v3/dtag_transfer_requests/{receiver}";
-  }
-
-  // Params queries the profiles module params
-  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
-    option (google.api.http).get = "/desmos/profiles/v3/params";
+    option (google.api.http).get = "/desmos/profiles/v3/dtag_transfer_requests";
   }
 
   // ChainLinks queries the chain links associated to the given user, if
   // provided. Otherwise it queries all the chain links stored.
   rpc ChainLinks(QueryChainLinksRequest) returns (QueryChainLinksResponse) {
-    option (google.api.http).get = "/desmos/profiles/v3/chain-links/{user}";
-  }
-
-  // UserChainLink queries the chain link for the given user, chain name and
-  // target address
-  rpc UserChainLink(QueryUserChainLinkRequest)
```

Comment:
Should we remain `UserChainLink` instead of `ChainLinks`? Allowing query all the chain links from the contracts or rpc might slow down the chain.

Reply:
Never mind, just figure out that not the problem since it increases the gas fee with the contract.